tools/UpdatePaths/Scripts/68565_qm_to_command.txt

```diff
@@ -0,0 +1 @@
+/area/command/heads_quarters/qm : /area/station/command/heads_quarters/qm
```

Comment:
Your updates path is effectively useless since I converted station areas from `/area/` to `/area/z-level`.

The way it should've been was `/area/station/cargo/qm` : `/area/station/commad/heads_quarters/qm`